### ISSUE TITLE
[BUG]Get-PnPTenantSite : The property or field 'AuthContextStrength' has not been initialized

### ISSUE THREAD

**Issue by nganbui** [pnp/powershell#931]
### Reporting an Issue or Missing Feature
Get-PnPTenantSite  getting The property or field 'AuthContextStrength' has not been initialized
### Actual behavior 
After connect to PnPOnline and run below cmdlet getting the error:
Get-PnPTenantSite -Url $siteUrl | select *
The property or field 'AuthContextStrength' has not been initialized

### Steps to reproduce behavior
Connect-pnpOnline
Get-PnPTenantSite -Url {siteUrl} | select *

### What is the version of the Cmdlet module you are running?
(you can retrieve this by executing ```Get-Module -Name "PnP.PowerShell" -ListAvailable```)

### Which operating system/environment are you running PnP PowerShell on?
- [x ] Windows
- [ ] Linux
- [ ] MacOS
- [ ] Azure Cloud Shell
- [ ] Azure Functions
- [ ] Other : please specify

**Comment by hicotton02** [comment]
I can confirm this. using the nightly build for last night:
`1.6.44-nightly       PnP.PowerShell                      PSGallery            Microsoft 365 Patterns and Practices`

i got this error:
`Get-PnPTenantSite : The property or field 'AuthContextStrength' has not been initialized. It has not been requested or the request has not been executed. It may need to be explicitly requested.
At line:1 char:9
+ $site = Get-PnPTenantSite -Url $SiteUrl`

**Comment by nganbui** [comment]
It seems still an issue. Anybody please help or any updates on this bug?
Thank you.

**Comment by veronicageek** [comment]
Have you closed and reopen your PowerShell console/terminal?

**Comment by nganbui** [comment]
@veronicageek Yes I did couple times but still have the issue as attached
![image](https://user-images.githubusercontent.com/7183509/126540475-1e0bc46f-45f7-43be-82d1-9f71e04370b5.png)

**Comment by veronicageek** [comment]
The issue seems resolved for everyone. I don't know your environment but try to only have the latest installed (1.6.44-nightly), restart your session, etc.

**Comment by gautamdsheth** [comment]
Looks like you have old version of module installed, please uninstall it first and then do a clean install and try again ?

**Comment by nganbui** [comment]
I am using version 1.6.0
@veronicageek  I need to update to 1.6.44. I think with prod environment should use GA major version. I will try with latest one to see if resolve the issue.
Thanks for help.

**Comment by nganbui** [comment]
Thanks everyone for help. It worked with 1.6.44 version but this is Prerelease which is not good on Prod environment.
@veronicageek  Do you know when it will be released?

**Comment by veronicageek** [comment]
I'll defer to @erwinvanhunen or @gautamdsheth to say when 1.7.0. will be coming out.

**Comment by kyle-herrmann** [comment]
I'm running version 1.6.0 and Get-PnPTenantSite throws this error on one UAT site and succeeds on another. I'm using the same system, module, and code. Seems like it may be a result of security settings in the tenant?

**Comment by mliben** [comment]
Oh dear. I've updated to version 1.6.45 and the problem seems to have returned. Opened in new PowerShell session :(

Are there dependencies that need to be updated?

get-module pnp.powershell

ModuleType Version   Name                                                                                                                                                                                                                                        
----------      -------    ----                                                                                                                                                                                                                                    
Manifest       1.6.45     PnP.PowerShell          


Get-PnPTenantSite : The remote server returned an error: (401) Unauthorized.
At line:37 char:1
+ Get-PnPTenantSite -IncludeOneDriveSites -Filter "url -like '-my.sharepoint.com/personal/'"
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : WriteError: (:) [Get-PnPTenantSite], WebException
    + FullyQualifiedErrorId : EXCEPTION,PnP.PowerShell.Commands.GetTenantSite

**Comment by ToddKlindt** [comment]
I started getting this a few days ago in an Azure Function, which uses 1.6.0. I have reproduced it in PowerShell with version 1.6.0 and 1.6.17. I have also gotten this authenticating with username and password and with ClientID and Certificate.
It feels to me like this is broken because something changed on the backend.

**Comment by nganbui** [comment]
> I started getting this a few days ago in an Azure Function, which uses 1.6.0. I have reproduced it in PowerShell with version 1.6.0 and 1.6.17. I have also gotten this authenticating with username and password and with ClientID and Certificate.
> It feels to me like this is broken because something changed on the backend.

Yes agree. I dont know if there is any fix for 1.6.0 but it seems to me still have same issue if using this version.

**Comment by veronicageek** [comment]
@nganbui - Your error message is not the same. So it hasn't "returned".
I have no issue on my side for your command in `1.6.45-nightly`.  Make sure you have the correct permissions or open a new issue with more details.

![image](https://user-images.githubusercontent.com/25181757/126682506-69577bab-5e93-4bf1-bfeb-99a7f7040c35.png)

I'm also closing this one because it has been solved, but you're waiting on a major release (`1.7.0`) which we have no ETA as of yet.

**Comment by nganbui** [comment]
> @nganbui - Your error message is not the same. So it hasn't "returned".
> I have no issue on my side for your command in `1.6.45-nightly`. Make sure you have the correct permissions or open a new issue with more details.
> 
> ![image](https://user-images.githubusercontent.com/25181757/126682506-69577bab-5e93-4bf1-bfeb-99a7f7040c35.png)
> 
> I'm also closing this one because it has been solved, but you're waiting on a major release (`1.7.0`) which we have no ETA as of yet.

Thanks. I am talking about the version 1.6.0 have the issue with this cmdlet. With 1.6.45-nightly worked for me.
